src/test/java/uk/gov/hmcts/reform/ccd/fixture/CaseDataEntityBuilder.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.ccd.fixture;
+
+import uk.gov.hmcts.reform.ccd.data.entity.CaseDataEntity;
+
+import java.time.LocalDate;
+import java.util.Optional;
+
+public class CaseDataEntityBuilder {
```

Comment:
Could this be achieved with a `@Builder` lombok annotation on `CaseDataEntity` ?